File: mobile/lib/features/operation_financiere/domain/entities/sous_categorie_operation.dart
class SousCategorieOperation {
  final int id;
  final String code;
  final String libelle;
  final int categorieId;
  final String? categorieLibelle;
  final bool actif;

  const SousCategorieOperation({
    required this.id,
    required this.code,
    required this.libelle,
    required this.categorieId,
    this.categorieLibelle,
    required this.actif,
  });
}

File: backend/src/main/java/com/tmk/vtcmanager/application/domain/operation/SousCategorieOperation.java
package com.tmk.vtcmanager.application.domain.operation;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SousCategorieOperation {

    private Long id;
    private String code;
    private String libelle;
    private Long categorieId;
    private boolean actif;
}
